src/cpp/src/tokenizer.cpp

```diff
@@ -22,39 +22,6 @@
 
 namespace {
 
-// todo: remove when openvino-tokenizers will support left padding
```

Comment:
We already have left padding in tokenizers

Reply:
should we remove this test then ? (which is skipped now, see CVS-159924) https://github.com/openvinotoolkit/openvino.genai/blob/523ea7c40667fd4c78ae6b0c817f32419a71db99/tests/python_tests/test_llm_pipeline.py#L454-L487 ?

Reply:
Yes, we can remove that test. Current tests cover more cases.